analysis_driver/pipelines/projects.py

```diff
@@ -1,14 +1,27 @@
 import os
 from egcg_core.util import find_file
+from egcg_core import clarity
 from analysis_driver.config import default as cfg, OutputFileConfiguration
-from analysis_driver.quality_control import Relatedness
+from analysis_driver.quality_control import Relatedness, Peddy, Genotype_gVCFs
 from analysis_driver.exceptions import PipelineError
 from analysis_driver.transfer_data import output_project_data
 from analysis_driver import segmentation
 
 
 def build_pipeline(dataset):
-    project_source = os.path.join(cfg.query('sample', 'delivery_source'), dataset.name)
+    project_id = dataset.name
+    sample_ids = [sample['sample_id'] for sample in dataset.samples_processed]
+
+    species_in_project = set()
+    for sample in dataset.samples_processed:
+        species = sample.get('species_name')
+        if not species:
+            species = clarity.get_species_from_sample(sample.get('sample_id'))
+        species_in_project.add(species)
+    if len(species_in_project) != 1:
+        raise PipelineError('Unexpected number of species (%s) in this project' % ', '.join(species_in_project))
+
+    project_source = os.path.join(cfg.query('sample', 'delivery_source'), project_id)
     gvcf_files = []
     for sample in dataset.samples_processed:
         gvcf_file = find_file(project_source, sample['sample_id'], sample['user_sample_id'] + '.g.vcf.gz')
```

Comment:
You're not adding the `gvcf_file` to `gvcf_files`. You should also check if you found gvcf_file